src/main/java/org/quantumbadger/redreader/activities/MainActivity.java

```diff
@@ -322,31 +322,53 @@ public void run() {
 
 				if(lastVersion <= 89) {
 					//Upgrading from 89/1.9.11 or lower, enable finer control over font scales
-					//and set them to match the existing settings and behavior
+					//and set them to match the existing settings
+					//The old Inbox Font Scale setting is ignored
 
 					final String existingPostFontscalePreference = Float.toString(PrefsUtility.appearance_fontscale_posts(this, sharedPreferences));
-
-					sharedPreferences.edit().putString(
-							getString(R.string.pref_appearance_fontscale_post_subtitles_key),
-							existingPostFontscalePreference
-					).apply();
-
-					sharedPreferences.edit().putBoolean(
-							getString(R.string.pref_appearance_fontscale_post_use_different_scales_key),
-							true
-					).apply();
-
-					final String existingCommentSelfTextFontscalePreference = Float.toString(PrefsUtility.appearance_fontscale_comments(this, sharedPreferences));
-
-					sharedPreferences.edit().putString(
-							getString(R.string.pref_appearance_fontscale_selftext_key),
-							existingCommentSelfTextFontscalePreference
-					).apply();
-
-					sharedPreferences.edit().putString(
-							getString(R.string.pref_appearance_fontscale_comment_headers_key),
-							existingCommentSelfTextFontscalePreference
-					).apply();
+					final String existingCommentSelfTextFontscalePreference = Float.toString(PrefsUtility.appearance_fontscale_bodytext(this, sharedPreferences));
```

Comment:
`Float.toString` might break here, because not all fractional numbers have a floating point representation (this may return something like 0.7999999999 instead of 0.8). We should get the actual preference string here rather than a float.